pymc3/tests/test_distributions_random.py

```diff
@@ -89,9 +89,16 @@ def pymc3_random(
 
 
 def pymc3_random_discrete(
-    dist, paramdomains, valuedomain=Domain([0]), ref_rand=None, size=100000, alpha=0.05, fails=20
+    dist,
+    paramdomains,
+    valuedomain=Domain([0]),
+    ref_rand=None,
+    size=100000,
+    alpha=0.05,
+    fails=20,
+    extra_args=None,
 ):
-    model = build_model(dist, valuedomain, paramdomains)
+    model = build_model(dist, valuedomain, paramdomains, extra_args=extra_args)
```

Comment:
This slipped in while I was trying to debug some test failures. Isn't actually salient to this PR.

Reply:
Cleaned.